docs/search/views/index.md

```diff
@@ -0,0 +1,317 @@
+
+The document vectors that are closer to a query vector are more relevant. The closeness is expressed as the cosine of the angle between two vectors ([cosine similarity](https://en.wikipedia.org/wiki/Cosine_similarity)). We evaluate the following expression to define how relevant document `d` is to query `q`:
+
+`cos a = (d * q) / (|d| * |q|)`
+
+* `d * q` is the dot product of the query vector `q` and document vector `d`
+* `|d|` is the norm of vector `d`
+* `|q|` is the norm of vector `q`
+
+The vector components must be computed up front. Since space is formed by terms, you can use *term weights* as coordinates. The following probability and statistical weighting models are implemented in C8Search:
+
+* [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25)
+* [TF-IDF](https://en.wikipedia.org/wiki/Tf%E2%80%93idf)
+
+Both models rely on two components:
+
+* *Term frequency* (TF): The number of times term `t` appears in document `d`.
+* *Inverse document frequency* (IDF): How common or rare the term is across all documents.
+
+Searching and ranking capabilities are provided by the [IResearch library](https://github.com/iresearch-toolkit/iresearch)
+
+## Integration
+
+To use collections as a data source, you must *link* them to a View. A link is a one-way data flow from a GDN collection to a Search View that determines how incoming data is made available to the user. A View can have multiple links that are each connected to a different collection, and each collection can be linked to multiple Views.
+
+To minimize performance reduction, we do not constantly synchronize Search Views as the linked collections change. You can change the consolidation policy to increase or decrease the rate at which Search Views synchronize with linked collections.
+
+You can link documents and edge collections to Search Views so your graphs can be treated as both flat and interconnected data structures. For example, you can find the most relevant vertices by searching and sorting with a View, then perform a regular tree search on each vertex.
+
+Edit the [View definition](#view-definition) to manage links between Views and collections. You can index any attribute at any depth, including nested attributes, and define Analyzers to process values for each field. To produce results, Analyzers you specify in the query must be defined in the View.
+
+By default, array elements are indexed individually as if each element is the value of the source attribute. You can use Analyzers to transform strings into multiple tokens that are handled similarly to an array of strings. Refer to [C8QL SEARCH operation](../c8ql/operations/search.md) for details. Primitive values other than strings (`null`, `true`, `false`, numbers) are indexed unchanged. You can choose to index nested object values under the respective attribute path, including objects in arrays.
+
+You can manage Views by using:
```

Comment:
In an ideal world, all three of these, plus the CLI, should have instructions in the task.